tests/basictests/distributed-workloads.sh

```diff
@@ -46,36 +46,105 @@ function install_distributed_workloads_kfdef(){
 
 function test_mcad_torchx_functionality() {
     header "Testing MCAD TorchX Functionality"
+
+    ########### Clean Cluster should be free of these resources ############
+    # Get appwrapper name
+    AW=$(oc get appwrapper -n ${ODHPROJECT} | grep mnistjob | cut -d ' ' -f 1) || true
+    # Clean up resources
+    if [[ -n $AW ]]; then
+        os::cmd::expect_success "oc delete appwrapper $AW -n ${ODHPROJECT} || true"
+    fi
+    os::cmd::expect_success "oc delete notebook jupyter-nb-kube-3aadmin -n ${ODHPROJECT} || true"
```

Comment:
@jbusche found that the pvc can carry over some junk file over so we should delete the pvc.
```suggestion
    os::cmd::expect_success "oc delete notebook jupyter-nb-kube-3aadmin -n ${ODHPROJECT} || true"
    os::cmd::expect_success "oc delete pvc jupyterhub-nb-kube-3aadmin-pvc -n ${ODHPROJECT} || true"
```